src/RemoteHandler/ServerMock.js

```diff
@@ -224,6 +224,8 @@ class ServerMock {
             this.responseText = JSON.stringify(ServerMock.getEventTypes());
         } else if(this.api.indexOf("events") > -1) {
             this.responseText = JSON.stringify(ServerMock.getEvents());
+        } else if(this.api.indexOf("create") > -1) {
+            this.responseText = JSON.stringify("");
         } else {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either merge this branch with the identical one on line "227" or change one of the implementations. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS1871)